architecture/faust/dsp/libfaust-box.h

```diff
@@ -648,4 +717,83 @@ LIBFAUST_API dsp_factory_base* createCPPDSPFactoryFromBoxes(const std::string& n
  @}
  */
 
+LIBFAUST_API bool isBoxAbstr(Box t);
```

Comment:
I think I need all of these functions to change in the same way inside `boxes/boxes.hh`

Reply:
Sorry, can you say more about the need/todo here?